example/starwars/starwars.go

```diff
@@ -572,8 +572,8 @@ func newFriendsConnectionResolver(ids []graphql.ID, args *friendsConenctionArgs)
 	if args.First != nil {
 		to = from + int(*args.First)
 	}
-	if to > len(ids)-from {
-		to = len(ids) - from
+	if to > len(ids) {
```

Comment:
Test and bugfix look good. Minor suggestion: Can we move this `if` one line up? Because it is only relevant if `args.First` is set.